### DISCUSSION TITLE
想请问这个报错是咋回事啊

### DISCUSSION THREAD

**Discussion by shiroyukki** [mikumifa/biliTickerBuy#213] [提问专区]
![UFGIY52YD~}%AR3X8)}FITX](https://github.com/mikumifa/biliTickerBuy/assets/72260642/7a489759-bd89-41b9-b476-dd5eafba4d92)
换了自己的服务器试了下也是同样的问题

**Comment by mikumifa** [comment]
报错搜一下，问问CSDN
